api.swagger.yaml

```diff
@@ -16736,6 +16736,21 @@ components:
         - parent_uuid
         - body
         - author_uuid
+    comment-author-schema:
+      properties:
+        uuid:
+          type: string
+          format: uuid
+          description: 'This is the Invotra UUID of the comment author'
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+        user_avatar:
+          type: string
+          description: The user's image
+          example: "https://dev.invotra.com/sites/default/files/styles/large_avatar/public/Dorothy%20Whitehead.jpg?itok=6p19"
+        display_name:
+          type: string
+          description: 'The display name of the comment author'
+          example: 'Gary Barlow'
```

Comment:
Why would Gary Barlow upload a picture of Dorothy Whitehead?

All jokes aside I think we should keep to the akagera names as that's what we do everywhere else in swagger

Take that Max